typescript/cosmos-sdk/src/hyperlane/interchain_security/query.ts

```diff
@@ -16,13 +16,19 @@ type QueryDecodedIsmsResponse = {
   pagination: pagination.PageResponse | undefined;
 };
 
+export const NoopIsmTypeUrl = '/hyperlane.core.interchain_security.v1.NoopISM';
+export const MerkleRootMultisigIsmTypeUrl =
+  '/hyperlane.core.interchain_security.v1.MerkleRootMultisigISM';
+export const MessageIdMultisigTypeUrl =
+  '/hyperlane.core.interchain_security.v1.MessageIdMultisigISM';
```

Comment:
This is just a personal preference, but I'd like this to be an enum instead of separate constants